dspot/src/main/java/fr/inria/diversify/utils/AmplificationHelper.java

```diff
@@ -203,6 +245,76 @@ public static CtMethod cloneMethodTest(CtMethod method, String suffix) {
         return cloned_method;
     }
 
+    /**
+     * Clones a test method for input-amplification purposes.
+     *
+     * Updates the counter of amplifications applied, assuming one cloning per amplification.
+     *
+     * See {@link #cloneMethodTest(CtMethod, String)} for the cloning details.
+     *
+     * @param method Method to be cloned
+     * @param suffix Suffix for the cloned method's name
+     * @return The cloned method
+     */
+    public static CtMethod cloneMethodTestIAmp(CtMethod method, String suffix) {
```

Comment:
We could call  cloneMethodTestIAmp(method, suffix, 1) rather than duplicate the code.